tchannel/serializer/json.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+import json
+
+from tchannel.schemes import JSON
+
+
+class JsonSerializer(object):
+    def __init__(self):
```

Comment:
BTW consider adding [`__slots__`](https://docs.python.org/2/reference/datamodel.html#slots) to the `Serializer` classes. Since we'll be instantiating these twice for each request/response (assuming we don't just make them stateless and re-use the same ones), we should avoid allocating dictionaries for each instance. `__slots__` will take care of that.